geoips/plugins/modules/procflows/config_based.py

```diff
@@ -764,21 +791,43 @@ def call(fnames, command_line_args=None):
         fnames = glob(config_dict["filenames"])
 
     output_file_list_fname = command_line_args["output_file_list_fname"]
+    reader_kwargs = None
     bg_files = None
     bg_product_name = None
+    bg_reader_kwargs = None
     bg_resampled_read = False
     bg_self_register_dataset = None
     bg_self_register_source = None
 
+    # 1. get reader_kwargs from command line if specified command line
+    # 2. next get reader_kwargs from YAML config_dict if specified within YAML config
+    # 3. Finally, if not specified elsewhere, default reader_kwargs to {}
+    if command_line_args.get("reader_kwargs"):
```

Comment:
Added both reader_kwargs and bg_reader_kwargs to the logic for setting appropriate variables.